client/src/components/DropSearch/DropSearch.tsx

```diff
@@ -89,7 +94,7 @@ class DropSearch extends Component<Props, State> {
    * - and pass either undefined or extracted object to prop (handleSearchChange method)
    * @returns undefined.
    */
-  handleSearchChange = (e: SyntheticEvent<HTMLButtonElement>) => {
+  handleSearchChange = e => {
```

Comment:
can not we add event type here?

Reply:
We don't need to add types to everything in our app. Mostly the model should be restricted with types. Including redux logic and component state.